### ISSUE TITLE
DataTable: table-style does not accept strings anymore

### ISSUE THREAD

**Issue by alwaqfi** [primefaces/primevue#5343]
### Describe the bug

VS Code gives this error `Type 'string' is not assignable to type 'object'.ts(2322)`.

![image](https://github.com/primefaces/primevue/assets/9500241/daddd967-1d16-45f8-bb1f-94c5c49b4c6e)

Examples/Sample code the DataTable section uses string 
Link: https://primevue.org/datatable/ => Basic 
```
<template>
    <div class="card">
        <DataTable :value="products" tableStyle="min-width: 50rem">
            <Column field="code" header="Code"></Column>
            <Column field="name" header="Name"></Column>
            <Column field="category" header="Category"></Column>
            <Column field="quantity" header="Quantity"></Column>
        </DataTable>
    </div>
</template>

<script setup>
import { ref, onMounted } from 'vue';
import { ProductService } from '@/service/ProductService';

onMounted(() => {
    ProductService.getProductsMini().then((data) => (products.value = data));
});

const products = ref();

</script>
```

This was introduced in version 3.48.1 and in the line below.
![image](https://github.com/primefaces/primevue/assets/9500241/c4ecf415-18a6-486f-92d8-bb66defdd83f)

### Reproducer

https://github.com/alwaqfi/prime-vue-data-table

### PrimeVue version

3.48.1

### Vue version

3.x

### Language

TypeScript

### Build / Runtime

Vite

### Steps to reproduce the behavior

1. Clone repo
2. Run npm install 
3. The property should show and error now

### Expected behavior

1. The property should accept string OR
2. Update documentation on how to use style props that now accept Objects

**Comment by lewandy** [comment]
I'm getting the same error, any luck about it?

**Comment by alwaqfi** [comment]
> I'm getting the same error, any luck about it?

Yes, I wish it was documented. Seems the new way to use this property is to convert it to an object  

```html
 <DataTable
....
        :table-style="{'min-width': '50rem', 'margin-top': '100px'}"
....
      >
</DataTable>
```

**Comment by lewandy** [comment]
Thank you very much

**Comment by Terrijoo** [comment]
If it really only accepts objects now, the documentation/examples (https://primevue.org/datatable) need to be adjusted too, since they also all use the string format still.
